assets/scss/moj-blocks-general.scss

```diff
@@ -7,25 +7,10 @@
 }
 
 .govuk-main-wrapper {
-  p:not([class]), p[class=""] {
```

Comment:
Did we check removing this didnt cause any list style issues?

Reply:
We did check and we decided that these styles should all be left in - so I have done that.  

This Hale element of the change is now far less important, but there are a few things.